lib/awful/client.lua

```diff
@@ -900,11 +900,11 @@ function client.setwfact(wfact, c)
     t:emit_signal("property::windowfact")
 end
 
---- Change a client's window factor.
+--- Change window factor of a client.
 --
 -- @legacylayout awful.client.incwfact
 -- @tparam number add Amount to increase/decrease the client's window.
---   Should be between 0.0 and 1.0.
+--   Should be between -1.0 and 1.0 (all window factors get normalized to 1).
```

Comment:
Isn't this still wrong? I would just drop this sentence.

As said in the other argument, the argument should be between `-current_window_factor` and `something close to infinite`. The normalisation then ensures that the sum of all factors is 1.